src/ifc-printer/printer.cxx

```diff
@@ -126,7 +126,7 @@ namespace ifc::util {
                     // use requested printing of index, we will print it as requested.
                     if (depth == 0 and implies(options, PrintOptions::Top_level_index)
                         and n.key.kind() != SortKind::Decl)
-                        stream << "-" << (int)n.key.index();
+                        stream << "-" << static_cast<int>(n.key.index());
```

Comment:
`ifc::to_underlying`

Reply:
Actually, `index()` returns a uint32_t already, the cast was pointless to begin with.